homeassistant/components/sensor/__init__.py

```diff
@@ -56,8 +62,17 @@
 
 DEVICE_CLASSES_SCHEMA = vol.All(vol.Lower, vol.In(DEVICE_CLASSES))
 
+# The state represents a measurement in present time
+STATE_CLASS_MEASUREMENT = "measurement"
+# The state does not fall inte any other defined state class
```

Comment:
There is no reason to do this, as we won't use the data anyway. Let's just expect people to return `None`.

Reply:
Fixed in cb50f9d